README.fr.md

```diff
@@ -80,6 +80,21 @@ dnf install make automake gcc gcc-c++ kernel-devel git ImageMagick libX11-devel
 
 </details>
 
+<details>
+
+<summary>Nix/Nixos</summary>
+
+<br>
+
+Installez toutes les dépendances grâce à:
```

Comment:
I think this is more adapted, to follow Fedora installation instructions style. It's just a suggestion : Ubuntu installation guide isn't the same that Fedora's one.
```suggestion
Installez toutes les dépendances grâce à cette commande:
```